app/src/main/res/values/strings.xml

```diff
@@ -66,5 +66,10 @@
     <string name="data_updated">Data updated!</string>
     <string name="saving_failed">Saving failed!</string>
     <string name="purchase_succeed">Purchase succeed!</string>
```

Comment:
@SergeiMikhailovskii my message about giving a meaningful name for localized resources is related to all of them in fact. You may leave it as is for now, but next time try to verify names of all of your resources and rename them accordingly.